client/txn.go

```diff
@@ -318,18 +317,11 @@ func (txn *Txn) Commit() error {
 
 // Rollback sends an EndTransactionRequest with Commit=false.
 func (txn *Txn) Rollback() error {
-	err := txn.sendEndTxnCall(false /* commit */)
-	// Explicitly set the status as ABORTED so that higher layers
-	// know that this transaction has ended.
-	txn.Proto.Status = proto.ABORTED
```

Comment:
note that this bit moved to sql/server.go. I'm not totally sure that's the right place for it. What do people think?